WordPress/twentytwentyfive
# Pattern review: CTA Pricing, clients and banner with description patterns.

**Description**

Partial to https://github.com/WordPress/twentytwentyfive/issues/302

- CTA Pricing pattern: Removed padding and used spacer instead.
- Clients: Removed padding and used spacer instead. Switched heading from H3 to H2 to match design.
- Banner with description: Set minimum column width to 26.

**Screenshots**

N/A

**Testing Instructions**

CTA pricing pattern

1. Create a page.
2. Use the CTA pricing pattern.
3. Confirm the padding is no longer there for the stack and we're using a spacer instead.

Clients pattern

1. Create a page
2. Use the clients pattern.
3. Confirm that we're no longer using padding and we are now using a spacer.
4. As a bonus, the H3 is now an H2, to respect the Figma design.

Banner with description

1. Create a page.
2. Use the "Banner with description and images grid"
3. Confirm that the grid minimum col width is set to 26.
